nomad/csi_endpoint_test.go

```diff
@@ -1995,7 +1988,7 @@ func TestCSIVolume_nodeExpandVolume(t *testing.T) {
 
 func TestCSIPluginEndpoint_RegisterViaFingerprint(t *testing.T) {
 	ci.Parallel(t)
-	srv, shutdown := TestServer(t, func(c *Config) {
+	srv, _, shutdown := TestACLServer(t, func(c *Config) {
```

Comment:
Note to reviewers: I have no idea why we were changing ACL config after startup for these CSI tests, but you can't actually do that and it's broken to even try now, so that's the only reason why these tests are changing.